playground-sveltekit/package.json

```diff
@@ -10,12 +10,12 @@
     "@storyblok/svelte": "^1.0.0",
     "@sveltejs/adapter-auto": "^2.1.0",
     "@sveltejs/kit": "^1.22.0",
-    "svelte": "^3.59.2",
+    "svelte": "^4.0.0",
     "svelte-check": "^3.4.4",
     "tslib": "^2.6.0",
-    "typescript": "^5.1.6",
+    "typescript": "^5.0.0",
```

Comment:
Any particular reason to fallback to `5.0.0`?

Reply:
sorry , my approach is to define the minimum version of the supported dependencies. I will update the version, according to the dependabot strategy (the latest version instead of the lower version)